csbuild/__init__.py

```diff
@@ -118,7 +118,7 @@ def _runFuncs(*args, **kwargs):
 				else:
 					with tempToolchain.Use(tool):
 						rets.append(func.__get__(tool)(*args, **kwargs))
-			if len(rets) == 1:
+			if len(rets) == 1: # pylint: disable=no-else-return
```

Comment:
Actually we should fix this one. The `elif` below this should just be another `if`.